### ISSUE TITLE
Investigate structured logging

### ISSUE THREAD

**Issue by jzohrab** [covidatlas/li#54]
**Description.

Currently devs use console.log with strings, which can be hard to parse and analyze.  Also, during dev, I often want to redirect logs etc.  Setting up some basic logging standards might be a good idea.

Refs:

* Structured logging: https://stackify.com/what-is-structured-logging-and-why-developers-need-it/
* And with Winston: https://stackify.com/winston-logging-tutorial/

In prod, winston could and should go to stdout, so it can be aggregated.  In dev, we might want this stuff going to a file, etc, for devs to debug.  Also, with better logging, we might be able to parallelize some things which are currently kept sequential for ease of reading logs.  (I know that this is not much of an issue for prod w/ Li due to the architecture, but for good testing and dev it might be helpful).

**Comment by camjc** [comment]
I find myself always `console.table`-ing the output of a source while I'm working on it, it'd be nice to do that always IMO (scraped successfully, heres the table of the output)

**Comment by jzohrab** [comment]
For "structured logging", I'd like to log things by json rather than by string, and include some base info everything (potentially source file ,source, some kind of date info, etc).  With struct logging, it's _much_ easier to parse logs, and if we use proper logging then for duv we could choose DEBUG, INFO, etc output, and redirect to files.  For prod, it would all go to console for cloudwatch.
